main.py

```diff
@@ -1,12 +1,243 @@
+            print("Creating a new customer record: \n")
+            name = input("What is the customer's name? ")
+            postal_code = input("What is the customer's postal code? ")
+            phone = input("What is the customer's phone number? ")
+            registered_at = input("When was the customer registered? ")
+            response = video_store.create_customer(name=name, postal_code=postal_code, phone=phone, registered_at=registered_at)
+
+            print_divider()
+            print("New customer record created: ", response["customer"])
+
+        elif choice == '7':
+            print(f"Updating customer record: {video_store.selected_video} \n")
+            name = input("What is the new customer's name? ")
+            postal_code = input("What is the new customer's postal code? ")
+            phone = input("What is the customer's new phone number? ")
+            response = video_store.update_video(name=name, postal_code=postal_code, phone=phone)
+
+            print_divider()
+            print("Updated customer:", response["customer"])
+
+        elif choice == '8':
+            video_store.delete_customer()
+
+            print_divider()
+            print("Customer record has been deleted.\n")
+
+            print_divider()
+            print(video_store.get_customers_list())
+
+        elif choice == '9':
+            select_by = input("Would you like to select by name or id: \n")
+            if select_by == "name":
+                title = input("What is the customer's name that you would like to select? ")
+                video_store.get_single_customer(title=title)
+            elif select_by == "id":
+                id = input("What is the customer's id that you would like to select? ")
+                if id.isnumeric():
+                    id = int(id)
+                    video_store.get_single_customer(id=id)
+            else:
+                print("Selection not valid. Please enter a customer's name or id.\n")
+            
+            if video_store.selected_video:
+                print_divider()
+                print("Selected video: ", video_store.selected_video)
+
+        elif choice == '10':
+            print_divider()
+            for customer in video_store.get_customers_list():
+                print(customer)
+
+##### RENTAL CHOICES #####
+        elif choice == '11':
+            print_divider()
+            print("Checking out a video to a customer: \n")
+
+            video_id = input("Please enter the video's id: ")
+            if video_id.isnumeric():
+                video_id = int(video_id)
+            video = video_store.check_out_video(video_id=video_id)
+            print(video)
+            
+            customer_id = input("Please enter the customer's id: ")
+            if customer_id.isnumeric():
+                customer_id = int(customer_id)
+            customer = video_store.check_out_video(customer_id=customer_id)
+            print(customer)
```

Comment:
The API and the VideoStore class expect the check_out to have both the video id and the customer id.  Calling the check_out_video function on line 194 will throw an error.  A working sequence would be:
1. get the video id from the user
2. get the customer id from the user
3. call check_out_video(customer_id, video_id)